packages/css/src/components/button/README.md

```diff
@@ -2,16 +2,16 @@
 
 # Button
 
-Allows the user to perform actions and operate the user interface.
+Allows the user to perform an action or operate the interface.
 
 ## Guidelines
 
-- Choose a short label that describes the function of the button.
-- Use the correct type of button for the corresponding application, e.g. `type="submit"` for the primary form button.
-- Make sure one can operate a button through a keyboard.
+- Describe the Button’s function through a short label in the infinitive form.
```

Comment:
Is there a simpler way to describe this? I've never heard of the infinitive form before 😅